### DISCUSSION TITLE
B Corp status

### DISCUSSION THREAD

**Discussion by codeinabox** [osmlab/name-suggestion-index#9842] [General]
I was thinking it could be quite useful to know if a brand has B Corp status - is this something that could be determined from Wikidata?

**Comment by 1ec5** [comment]
In theory, yes, Wikidata has this information, though coverage is currently hit or miss. The structure looks something like this:

* A brand could be dual-tagged as a business or could be [owned by <small>(P127)</small>](https://www.wikidata.org/wiki/Property:P127) the business.
* The business may be have a [legal form <small>(P1454)</small>](https://www.wikidata.org/wiki/Property:P1454) of [benefit corporation <small>(Q4887441)</small>](https://www.wikidata.org/wiki/Q4887441), but technically B Lab allows even a C corporation to state that it [has certification <small>(P10611)</small>](https://www.wikidata.org/wiki/Property:P10611) [B Corporation <small>(Q16243343)</small>](https://www.wikidata.org/wiki/Q16243343).

Putting it all together, you can query for B Corp–owned brands using the [Wikidata Query Service](https://w.wiki/B2ij) or [QLever](https://qlever.cs.uni-freiburg.de/wikidata/aSqnBT).

I wouldn’t recommend indexing this information directly in NSI, since it’s so tangential to the identity of a mappable brand.
